eng/docker/libraries-sdk.linux.Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Builds and copies library artifacts into target dotnet sdk image
 ARG BUILD_BASE_IMAGE=mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9
-ARG SDK_BASE_IMAGE=mcr.microsoft.com/dotnet/nightly/sdk:8.0
+ARG SDK_BASE_IMAGE=mcr.microsoft.com/dotnet/nightly/sdk:latest
```

Comment:
do we need to worry about cases when "latest" != VERSION

Reply:
not really, we download the nightly SDK anyway for build purposes (there is no docker image for that). "latest" should guarantee we have all necessary prerequisites (I didn't want to use runtime-deps image because it does not have wget)